[2020-07-22 11:37:33,577 @run_system.py:76] INFO: scene: G:/dataset/pointCloud/data/ownTrainedData/test\2_aa_raw.pcd
[2020-07-22 11:37:34,124 @run_system.py:79] INFO: scene point number (5286147, 6)
